.eslintrc.js

```diff
@@ -0,0 +1,17 @@
+module.exports = {
+  root: true,
+  env: {
+    node: true
+  },
+  extends: [
+    'plugin:vue/essential',
+    '@vue/standard'
+  ],
+  parserOptions: {
+    parser: 'babel-eslint'
+  },
+  rules: {
+    'no-console': process.env.NODE_ENV === 'production' ? 'warn' : 'off',
+    'no-debugger': process.env.NODE_ENV === 'production' ? 'warn' : 'off'
+  }
+}
```

Comment:
Looks like we'll want to add our own rules to this `.eslintrc` with whatever rules we want. Could snag the rules from the `eslintrc` we used in mod 2?

Reply:
this project is using a standard eslinter, so it's pretty much already set up.

Reply:
What rules did you want to add, though?

Reply:
Oh maybe I'm confused. I thought it didn't really have any rules yet. I was thinking about the usual things like indentation, etc.